src/api/extension/index.js

```diff
@@ -1297,6 +1297,7 @@ export const initHW = async ({ device, id }) => {
  * @param {string} assetName utf8 encoded
  */
 export const getAdaHandle = async (assetName) => {
+  if (!assetName || assetName.length == 0) return null;
   const network = await getNetwork();
   const assetNameHex = Buffer.from(assetName).toString('hex');
```

Comment:
Thanks for the PR! Maybe it's better to move the if statement behind `assetNameHex` and check if this is not empty, because this is the variable that is concatenated with the policy. Just to make 100% sure.

Reply:
Good though, I'll make the change.